packages/react/src/components/Authenticator/Router/index.tsx

```diff
@@ -11,15 +11,17 @@ import { ConfirmVerifyUser, VerifyUser } from '../VerifyUser';
 import { ConfirmSignIn } from '../ConfirmSignIn/ConfirmSignIn';
 import { ConfirmResetPassword, ResetPassword } from '../ResetPassword';
 
+type AuthenticatorRenderProp = ({
+  signOut,
+  user,
+}: {
+  signOut: ReturnType<typeof useAuthenticator>['signOut'];
+  user: CognitoUserAmplify;
```

Comment:
Thinking maybe we can go ahead and make these optional as well since it will not be a breaking change

Reply:
+1!

Reply:
```suggestion
  signOut?: ReturnType<typeof useAuthenticator>['signOut'];
  user?: CognitoUserAmplify;
```